paasta_tools/check_marathon_services_replication.py

```diff
@@ -198,12 +193,13 @@ def send_event_if_under_replication(
     service,
     instance,
     cluster,
-    crit_threshold,
     expected_count,
     num_available,
     soa_dir,
 ):
     full_name = compose_job_id(service, instance)
+    job_config = marathon_tools.load_marathon_service_config(service, instance, cluster)
```

Comment:
Honestly I think it might be a little too messy to add another parameter to the various functions here. It wouldn't be the end of the world but this check adds no overhead over what was there before